app/controllers/users/invitations_controller.rb

```diff
@@ -1,13 +1,25 @@
 class Users::InvitationsController < Devise::InvitationsController
-  before_action :authorise_manage_team, only: %i(create new)
   before_action :delete_user_record, if: :user_should_be_cleared?, only: :create
-  before_action :add_organisation_to_params, :validate_invited_user, only: :create
-
+  before_action :add_organisation_to_params, only: :create
 
 private
 
+  def authenticate_inviter!
```

Comment:
I'm guessing this is a Devise method that we are overriding. It would be good to link directly to the definition of this method in a comment, similar how we've done it on L53 below.